classes/submission/reviewRound/ReviewRoundDAO.inc.php

```diff
@@ -218,13 +218,14 @@ function getCurrentRoundBySubmissionId($submissionId, $stageId = null) {
 	function getLastReviewRoundBySubmissionId($submissionId, $stageId = null) {
 		$params = array((int)$submissionId);
 		if ($stageId) $params[] = (int) $stageId;
+		$isSqlServer = Config::getVar('database', 'ms_sql');
 		$result = $this->retrieve(
 			'SELECT	*
 			FROM	review_rounds
 			WHERE	submission_id = ?
 			' . ($stageId ? ' AND stage_id = ?' : '') . '
 			ORDER BY stage_id DESC, round DESC
-			LIMIT 1',
+			' . ($isSqlServer ? 'OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY' : 'LIMIT 1'),
```

Comment:
As above, use `DAO::retrieveRange` for all DB types rather than add a SQL Server specific consideration.